editor/src/utils/feature-switches.ts

```diff
@@ -33,6 +35,7 @@ let FeatureSwitches: { [feature in FeatureName]: boolean } = {
   'Debug – Performance Marks (Slow)': false,
   'Debug – Performance Marks (Fast)': false,
   'Debug – Measure Selectors': false,
+  'Selectors Split': true,
```

Comment:
Is this used? I don't recall seeing it, so I'm guessing it can now be deleted

Reply:
ahhhh good find!

Reply:
fixed